Changed_copyfileslab4/copyfileslab4/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace copyfileslab4
+{
+    internal class Program
+    {
+        static void Main(string[] arguments)
+        {
+            if (arguments.Length < 3)
+            {
+                Console.WriteLine("You wrote less than 3 arguments");
+                Console.WriteLine("An example of the correct launch of the program: copyfileslab4.exe \"D:\\Documents\" \"D:\\Games\" \"*.txt\" \"*.pptx\"");
+                Console.WriteLine("Program execution did not complete successfully");
```

Comment:
take a look at Environment.ExitCode

Reply:
Added to the program Environment.Exitcode = 1 if there is an error and Environment.Exitcode = 0 if everything is fine.